utils/pagerduty_api.py

```diff
@@ -25,14 +29,22 @@ def get_pagerduty_users(self, resource_type, resource_id):
 
         return users
 
+    def get_user(self, user_id):
+        for user in self.users:
+            if user.id == user_id:
```

Comment:
What's this `id`? I cannot find it in https://developer.pagerduty.com/api-reference/reference/REST/openapiv3.json/components/schemas/User

Reply:
`id` is a user's ID in PagerDuty. example: `PV1WLMF` - https://redhat.pagerduty.com/users/PV1WLMF